pkg/apis/wavefront/v1alpha1/wavefrontproxy_types.go

```diff
@@ -22,6 +22,12 @@ type WavefrontProxySpec struct {
 	// Wavefront API Token.
 	Token string `json:"token"`
 
+	// This flag is specifically for Openshift, this need to be set to true when running proxy in Openshift platform.
```

Comment:
nit: Lets move all openshift related spec to the bottom to better distinguish it.

Reply:
Done